supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/CondorBackendFactory.scala

```diff
@@ -0,0 +1,38 @@
+package cromwell.backend.impl.htcondor
+
+
+import akka.actor.Props
+import com.typesafe.config.Config
+import cromwell.backend._
+import cromwell.core.CallContext
+import wdl4s.Call
+import wdl4s.expression.WdlStandardLibraryFunctions
+
+case class CondorBackendFactory(config: Config) extends BackendLifecycleActorFactory {
+  override def workflowInitializationActorProps(workflowDescriptor: BackendWorkflowDescriptor,
+                                                calls: Seq[Call],
+                                                configurationDescriptor: BackendConfigurationDescriptor): Option[Props] = {
+    Option(CondorInitializationActor.props(workflowDescriptor, calls, configurationDescriptor))
```

Comment:
Just saw it..it's doing some validations for the runtime attributes.. I suggest we leave this as is.